### DISCUSSION TITLE
Navigator di Flask Dashboard

### DISCUSSION THREAD

**Discussion by shinddayo** [teamalgoritma/community#445] [Q&A] [answered]
Mohon bantuannya untuk membuat menu, navigator di flask dashboard python

**Comment by kevbow** [comment, marked as answer]
Dear @shinddayo ,

Apakah yang dimaksud adalah navigation bar? Jika iya, berikut kami lampirkan referensi untuk contoh penggunaan library Flask-Navigation:
- [Dokumentasi resmi](https://flask-navigation.readthedocs.io/en/latest/)
- [Contoh dari geeksforgeeks](https://www.geeksforgeeks.org/navigation-bars-in-flask/)

Demikian yang dapat kami sampaikan, semoga membantu.
